src/oca_github_bot/manifest.py

```diff
@@ -199,3 +199,26 @@ def get_odoo_series_from_version(version):
     if not series:
         raise OdooSeriesNotDetected()
     return tuple(int(s) for s in series.split("."))
+
+
+def _user_can_push(gh, org, repo, username, addons_dir, target_branch):
```

Comment:
why this function begins with "_" ? the others function don't.

Reply:
It's a rename of `_user_can_merge`. And why `_user_can_merge` begins with "_"? I don't know. But you can ask to @sbidoul